Sources/BSWInterfaceKit/Extensions/UIContentView+Utilities.swift

```diff
@@ -47,4 +53,21 @@ public class ContentHeaderView<T: UIContentConfiguration>: UICollectionReusableV
     }
 }
 
+public class ContentFooterView<T: UIContentConfiguration>: UICollectionReusableView {
```

Comment:
I think you can rename and reuse the `ContentHeaderView ` to `ContentReusableView `